app/views/errors.py

```diff
@@ -30,46 +30,46 @@ def add_cache_control(response):
 @errors_blueprint.app_errorhandler(401)
 @errors_blueprint.app_errorhandler(NoTokenException)
 def unauthorized(error=None):
-    log_exception(error)
+    log_exception(error, 401)
     return render_theme_template('default', 'session-expired.html'), 401
 
 
 @errors_blueprint.app_errorhandler(InvalidTokenException)
 def forbidden(error=None):
-    log_exception(error)
+    log_exception(error, 403)
     return _render_error_page(403)
 
 
 @errors_blueprint.app_errorhandler(404)
 @errors_blueprint.app_errorhandler(QuestionnaireException)
 def page_not_found(error=None):
-    log_exception(error)
+    log_exception(error, 404)
     return _render_error_page(404)
 
 
 @errors_blueprint.app_errorhandler(SubmissionFailedException)
 def service_unavailable(error=None):
-    log_exception(error)
+    log_exception(error, 503)
     return _render_error_page(503)
 
 
 @errors_blueprint.app_errorhandler(MultipleSurveyError)
 def multiple_survey_error(error=None):
-    log_exception(error)
+    log_exception(error, 200)
```

Comment:
This PR isn't really about changing the status that is returned. Its more adding disability of what is returned. @ajmaddaford what do you think about this?